ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/CVEsTable.tsx

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 import { gql } from '@apollo/client';
 import {
+    ActionsColumn,
+    ExpandableRowContent,
     TableComposable,
     Tbody,
     Td,
     Th,
     Thead,
     Tr,
-    ExpandableRowContent,
 } from '@patternfly/react-table';
-import { Button, ButtonVariant, Text } from '@patternfly/react-core';
+import { Button, ButtonVariant, Text, pluralize } from '@patternfly/react-core';
```

Comment:
What are pro and con to import `pluralize` from PatternFly versus from `'pluralize'` package?

Reply:
Oh, I see: it returns count and noun, instead of more verbose template literal elsewhere.

Reply:
Yes what you said, but also I tend to gravitate towards that since the chances of us removing `pluralize` as a dependency is much, much greater than the chances we remove PatternFly in the future.